core-primitives/types/Cargo.toml

```diff
@@ -22,6 +22,7 @@ substrate-api-client = { default-features = false, features = ["sync-api"], git
 # substrate-deps
 frame-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
 frame-system = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
+pallet-assets = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
```

Comment:
Maybe we should introduce zepter here ;)

Reply:
yes...